modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -2394,71 +2390,44 @@ public synchronized void cleanupWorkflowInstances(int buffer, WorkflowState stat
 
   @Override
   public void repopulate(final String indexName) throws Exception {
-    List<Job> jobs = new ArrayList<>();
-    try {
-      for (Job job : serviceRegistry.getJobs(WorkflowService.JOB_TYPE, null)) {
-        if (WorkflowServiceImpl.Operation.START_WORKFLOW.toString().equals(job.getOperation())) {
-          jobs.add(job);
-        }
-      }
-    } catch (ServiceRegistryException e) {
-      logger.error("Unable to load the workflows jobs: {}", e.getMessage());
-      throw new ServiceException(e.getMessage());
-    }
-
-    final String destinationId = WorkflowItem.WORKFLOW_QUEUE_PREFIX + WordUtils.capitalize(indexName);
-    if (jobs.size() > 0) {
-      logger.info("Populating index '{}' with {} workflows", indexName, jobs.size());
-      final int total = jobs.size();
+    List<String> workflows =  serviceRegistry.getJobPayloads(Operation.START_WORKFLOW.toString());
+
+    final String destinationId = WorkflowItem.WORKFLOW_QUEUE_PREFIX + indexName.substring(0, 1).toUpperCase()
+            + indexName.substring(1);
+    logger.warn("index name: {}", destinationId);
```

Comment:
Should this be warn?  Or debug?  Caps should be fixed regardless.

Reply:
I think this is a left-over from testing. This can be removed.